packaging/rpm/cvmfs-universal.spec

```diff
@@ -83,6 +83,13 @@ BuildRequires: pkgconfig
 BuildRequires: python-devel
 BuildRequires: unzip
 
+%if 0%{?el4} || 0%{?el5} || 0%{?el6}
```

Comment:
I guess SLESS11/12 and possibly openSuSE 13 are also python2 platforms, at least to be checked